go/service/team_handler.go

```diff
@@ -151,6 +155,50 @@ func (r *teamHandler) abandonTeam(ctx context.Context, cli gregor1.IncomingInter
 	return nil
 }
 
+func (r *teamHandler) findAndDismissResetBadges(ctx context.Context, cli gregor1.IncomingInterface, teamName string) error {
+	badges := r.badger.State().FindResetMemberBadges(teamName)
+	if len(badges) == 0 {
+		return nil
+	}
+	r.G().Log.CDebugf(ctx, "Checking reset badges: got total %d badges for team %q",
+		len(badges), teamName)
+
+	team, err := teams.GetMaybeAdminByStringName(ctx, r.G(), teamName, false /* public */)
+	if err != nil {
+		return err
+	}
+
+	for _, badge := range badges {
+		var dismiss bool
+		teamUV, notFoundErr := team.UserVersionByUID(ctx, badge.Uid)
+		if notFoundErr == nil {
+			loadUserArg := libkb.NewLoadUserByUIDArg(ctx, r.G(), badge.Uid).WithPublicKeyOptional()
+			u, err := libkb.LoadUser(loadUserArg)
```

Comment:
Use the UPAK loader here.

Reply:
You might need the forceRepoll flag here.

Reply:
done, thanks!